lib/cisco_node_utils/cmd_ref/stp_global.yaml

```diff
@@ -78,7 +78,8 @@ bridge_assurance:
   default_value: true
 
 domain:
-  _exclude: [N3k, N9k-F, N9k]
+  _exclude: [N9k-F]
+  os_version: 'N3k:7.0.3.I6.1, N9k:7.0.3.I6.1'
```

Comment:
It would be nice if we could support either of the following:
`os_version: 'N3k:7.0.3.I6.1, N9k:7.0.3.I6.1'`
or
`os_version: 'N3k, N9k:7.0.3.I6.1`